contracts/merkle/SuperMerkleDistributor.sol

```diff
@@ -0,0 +1,230 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.7;
+
+import "../base/Sweepable.sol";
+import "../ISuper1155.sol";
+
+/**
+  @title A merkle distributor.
+  @author Qazawat Zirak
+
+  This contract replaces the traditional whitelists by using a merkle 
+  tree, storing the root on-chain instead of all the addressses. The 
+  merkle tree alongside the whitelist is kept off-chain for lookups and 
+  creating proofs to validate a claim.
+  This code is inspired by and modified from incredible work from RicMoo.
+  https://github.com/ricmoo/ethers-airdrop/blob/master/AirDropToken.sol
+
+  October 12th, 2021.
+*/
+contract SuperMerkleDistributor is Sweepable {
```

Comment:
Can't help but notice there's a lot of duplicate code between these two contracts. Can we see `SuperMerkleDistributor` implemented by inheriting from `SuperMerkleAccess`? Ultimately that's how we want to be using this thing.